lib/event.rb

```diff
@@ -0,0 +1,38 @@
+class Event
+  attr_reader :name, :ages
+
+  def initialize(name, ages)
+    @name = name
+    @ages = ages
+  end
+
+  def max_age
+    p @ages.sort[-1]
```

Comment:
You don't need the `p` here. `p` and `puts` and `print` are for printing to the screen. All we need here is to return the correct value